### PR TITLE
Adding parameter to disable JS minification to point to a quick worka…

### PR DESCRIPTION
**Pull request by yohhaan**
Hello!

Just a quick change to add to the `config.toml` file a reference to a workaround to an issue with the current version of the starter template. The issue is due to the JS minification process causing the dropdown menu not behaving anymore as responsive. 

I wished that these few lines had been in the file when I cloned the starter repo as this would have saved me some time trying to figure out a solution (I was almost at the point of switching completely of template/theme as this was not working, which is a shame as this is a cool template). So, decided to submit this PR in case this happens to anyone else. 

Best,

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -77,6 +83,7 @@ lastmod = ["lastmod", ":git", "date"]
 
 [[menu.main]]
   url = "#contact"
+
```
Comment (stp-ip) [inline config.toml:86]: This whitespace seems unnecessary here and most likely was leftover from your custom config I assume.

**Comment by stp-ip** [comment]
Thank you for taking the time to create the PR. I agree this is definitely helpful and open to more contributions in the future, if anything comes up.

File: config.toml [hunk 2]
```diff
@@ -62,6 +62,12 @@ lastmod = ["lastmod", ":git", "date"]
     light = "#f8f9fa"
     dark = "#343a40"
 
+# Disable JS minification by turning the following flag to true
+# Workaround that solves the issue of the dropdown menu not being responsive
```
[the remark this answers is not in the document]
Comment (yohhaan) [reply config.toml]: True, I have edited the comment on the following commit 2a52b541de718a1c30bfbd8af05a94e9f3943318
Feel free to change it as you see fit.

File: config.toml (hunk 1 above)
Comment (yohhaan) [reply config.toml]: oups deleted that on the new commit

**Comment by yohhaan** [comment]
Submitted a new commit (2a52b541de718a1c30bfbd8af05a94e9f3943318) with an edit of the comment and deletion of that new line/empty space, thanks for your time and effort on creating this cool template.

**Review by stp-ip — APPROVED** [review]
Great thank you.
/lgtm

**Pull request merged**